docs/application/web/guides/connectivity/bluetooth.md

```diff
@@ -697,8 +757,531 @@ To read and write a value of the Bluetooth descriptor:
+   ```
+   var setCallbackSuccessCB() {
+      console.log("Callback set successfully!");
+   };
+
+   var setCallbackErrorCB = function(error) {
+      console.error("Callback is not set correctly, error: " + error.name + "; " + error.message);
+   };
+
+   var sendResponseSuccessCB() {
+      console.log("Response sent successfully!");
+   };
+
+   var sendResponseErrorCB = function(error) {
+      console.error("Response send failure, error: " + error.name + "; " + error.message);
+   };
+   ```
+
+5. To register the callback called when a client reads the value of the characteristic from the local GATT server, create the callback function and pass it as an argument to the `setReadValueRequestCallback():
+   ```
+   var characteristicReadRequestCallback = function(clientAddress, offset) {
+      console.log(clientAddress + " requested to read characteristic's value with offset: " + offset);
+      return new tizen.GATTRequestReply(0, "0x1234");
+   };
+   server.services[0].characteristics[0].setReadValueRequestCallback(characteristicReadRequestCallback, setCallbackSuccessCB, setCallbackErrorCB, sendResponseSuccessCB, sendResponseErrorCB);
+   ```
+
+   > **Note**
+   > A callback set with `setReadValueRequestCallback()` function overwrites any previously set `ReadValueRequestCallback` on this characteristic.
+
+6. To register the callback called when a client writes a value of the characteristic of the local GATT server, create the callback function and pass it as an argument to the `setWriteValueRequestCallback()` method called on the characteristic object.
+   ```
+   var characteristicWriteRequestCallback = function(clientAddress, value, offset, replyRequired) {
+      console.log(clientAddress + " requested to write characteristic's value: " + value +
+                  " with offset: " + offset);
+      return replyRequired ? new tizen.GATTRequestReply(0) : null;
+   };
+
+   server.services[0].characteristics[0].setWriteValueRequestCallback(characteristicWriteRequestCallback, setCallbackSuccessCB, setCallbackErrorCB, sendResponseSuccessCB, sendResponseErrorCB);
+   ```
+
+   > **Note**
+   > A callback set with `setWriteValueRequestCallback()` function overwrites any previously set `WriteValueRequestCallback` on this characteristic.
+
+7. To register the callback called when a client reads the value of the descriptor from the local GATT server, create the callback function and pass it as an argument to the `setReadValueRequestCallback()` method called on the descriptor object.
+   ```
+   var descriptorReadRequestCallback = function(clientAddress, offset) {
+      console.log(clientAddress + " requested to read descriptor's value with offset: " + offset);
+      return new tizen.GATTRequestReply(0, "0x1234");
+   };
+   server.services[0].characteristics[0].descriptors[0].setReadValueRequestCallback(characteristicReadRequestCallback, setCallbackSuccessCB, setCallbackErrorCB, sendResponseSuccessCB, sendResponseErrorCB);
```

Comment:
Question. According to the line 1218, it seems the descriptorReadRequestCallback of line 1220 should be the argument, but characteristicReadRequestCallback is. Is it ok?

Reply:
Done